translations/en-us.yml

```diff
@@ -1326,7 +1326,25 @@ registries:
                 invalidProviderLink: 'See all files in this registration.'
                 osfstorage: 'OSF Storage'
         resources:
-            title: Resources
+            title: 'Resources'
+            left_nav_title: 'Open practice resources'
```

Comment:
```suggestion
            left_nav_title: 'Open resources'
```

Reply:
Hmm, my code is already changed to this. Let me see why the change is not reflected here as well.

Reply:
This is not per the mockups, though does fit on one line. I have tried both versions and see why Open practice resources were used. We can discuss more later, but do prefer this way.